src/assets/teamInfo.json

```diff
@@ -0,0 +1,52 @@
+[
```

Comment:
i think we should move this elsewhere and make a call to it so that way when we get new people we dont have to make a PR, we can just have them edit a google sheet or something!